src/libraries/System.Console/tests/ReadAndWrite.cs

```diff
@@ -316,6 +316,13 @@ public static void OutputEncoding_Browser()
         Assert.Throws<PlatformNotSupportedException>(() => Console.OutputEncoding = curEncoding );
     }
 
+    [Fact]
+    [PlatformSpecific(TestPlatforms.Browser)]
+    public static void InputEncoding_Browser()
```

Comment:
I'd use a different suffix than `_Browser`... presumably there will be other platforms (iOS? Android? etc.) where such members throw.  I think it'd be better to just state what's being tested, e.g. `InputEncoding_Throws_PlatformNotSupportedException`, and it's then just enabled on the relevant platforms.

Reply:
Made test names more specific